src/components/icon/icon.styles.jsx

```diff
@@ -0,0 +1,20 @@
+import { Tooltip } from 'react-tooltip';
```

Comment:
I think we should stick to one styling library either use styles component or sass